kouncil-backend/src/main/java/com/consdata/kouncil/track/TrackController.java

```diff
@@ -27,8 +28,11 @@
 @RestController
 public class TrackController extends AbstractMessagesController {
 
-    public TrackController(KafkaConnectionService kafkaConnectionService) {
+    private final SimpMessagingTemplate template;
```

Comment:
Lepiej jakoś bym to nazwał niż "template" ;)

Reply:
oki